### ISSUE TITLE
Integration testing with tmt

### ISSUE THREAD

**Issue by frantisekz** [teemtee/testcloud#3]
Copy out and run integration tests with tmt on every PR and commit.

**Comment by psss** [comment]
Yes, this would be very nice. Enabling package building would also allow us to test with fresh testcloud copr build from pull requests.

**Comment by psss** [comment]
Yes, we need to enable testing for pull requests as soon as possible. Adding to `Proposed`.
